rc/base/lint.kak

```diff
@@ -41,7 +41,7 @@ def lint -docstring 'Parse the current buffer with a linter' %{
                 }
             }
             /:[0-9]+:[0-9]+:/ {
-                errors = errors $2 "," $3 "," substr($4,2) ":"
+                errors = errors $2 "," $3 ",c" $3 " " substr($4,2) ":"
```

Comment:
So, we just prepend 'c<column> ' in front of the error description ? that `c` looks strange, but I dont have any totally satisfying alternative, maybe `:<column> ` would be closer to the traditional error format.